src/main/java/com/exactpro/th2/common/schema/filter/strategy/impl/AbstractFilterStrategy.java

```diff
@@ -64,12 +64,9 @@ private boolean checkValues(Map<String, String> messageFields, MultiValuedMap<St
     }
 
     private boolean checkValue(String value, FieldFilterConfiguration filterConfiguration) {
-        if (StringUtils.isEmpty(value)) {
-            return false;
-        }
-
         var valueInConf = filterConfiguration.getExpectedValue();
 
+        // FIXME: Change switch to switch-expression after upping java version
         switch (filterConfiguration.getOperation()) {
             case EQUAL:
                 return value.equals(valueInConf);
```

Comment:
Shouldn't we check the value for `null`? Probably, we should use save equals (`Objects.equals(value, valueInConf)`)